addon/services/intl.js

```diff
@@ -84,15 +84,19 @@ export default Service.extend(Evented, {
   /** @private **/
   _timer: null,
 
+  /** @private **/
+  _formats: null,
+
   /** @private **/
   _formatters: null,
 
-  _intls: null,
+  /** @private */
+  _intls: {},
```

Comment:
This Object is hanging off the proto, can be set in the constructor instead and then don't have to worry about leaking intl objects between instances of the service which I believe may only happen in testing.